src/org/jmc/gui/ExportWindow.java

```diff
@@ -745,6 +746,7 @@ private void sendExport() {
 
 				};
 
+				jfc.setSelectedFile(new File("default.obj"));

```

Comment:
I think this would be better as minecraft.obj
```suggestion
				jfc.setSelectedFile(new File("minecraft.obj"));
```